### PR TITLE
Fix grid bug with run_basic_inference

### LINKED ISSUE #62: 🐛[BUG]: README example not working with e2mip://fcn
### Version

main

### On which installation method(s) does this occur?

Pip

### Describe the issue

I get this bug

```
In [3]: >>> import datetime
   ...: >>> from earth2mip.networks import get_model
   ...: >>> from earth2mip.initial_conditions import cds
   ...: >>> from earth2mip.inference_ensemble import run_basic_inference
   ...: >>> time_loop  = get_model("e2mip://fcn", device="cuda:0")
   ...: >>> data_source = cds.DataSource(time_loop.in_channel_names)
   ...: >>> ds = run_basic_inference(time_loop, n=10, data_source=data_source, time=datetime.datetime(2018, 1, 1))
   ...: >>> ds.chunk()

File ~/.local/lib/python3.10/site-packages/modulus/models/afno/afno.py:372, in PatchEmbed.forward(self, x)
    369 def forward(self, x: Tensor) -> Tensor:
    370     B, C, H, W = x.shape
    371     assert (
--> 372         H == self.img_size[0] and W == self.img_size[1]
    373     ), f"Input image size ({H}*{W}) doesn't match model ({self.img_size[0]}*{self.img_size[1]})."
    374     x = self.proj(x).flatten(2).transpose(1, 2)
    375     return x

AssertionError: Input image size (721*1440) doesn't match model (720*1440).
````

this is probably a bug in run_basic_inference.

### Environment details

_No response_

### PR DESCRIPTION
**Pull request by nbren12**
Resolves #62

# Earth-2 MIP Pull Request

## Description

Resolves #runb

## Checklist

- [x] I am familiar with the Contributing Guidelines.
- [x] New or existing tests cover these changes.
- [x] The documentation is up to date with these changes.
- [x] The CHANGELOG.md is up to date with these changes.
- [x] An [issue](https://github.com/NVIDIA/earth2mip/issues) is linked to this pull request.

### CODE REVIEW

File: tests/test_end_to_end.py [hunk 1]
```diff
@@ -129,3 +132,11 @@ def test_inference_medium_range(tmpdir, regtest):
         metrics.rmse, digits=3
     )
     metrics.info(regtest)
+
+
+@pytest.mark.dlow
```
Comment (NickGeneva) [inline tests/test_end_to_end.py:137]: dlow or slow?

File: earth2mip/regrid.py [hunk 2]
```diff
@@ -59,6 +59,11 @@ def forward(self, x):
         return x
 
 
+class DropLast(torch.nn.Module):
```
Comment (NickGeneva) [inline earth2mip/regrid.py:62]: Nit: DropLastLat? Or something to indicate latitude thats dropped here. Non-blocking

**Review by NickGeneva — APPROVED** [review]

File: tests/test_end_to_end.py (hunk 1 above)
Comment (nbren12) [reply tests/test_end_to_end.py:137]: fixed

File: earth2mip/regrid.py (hunk 2 above)
Comment (nbren12) [reply earth2mip/regrid.py:62]: Renamed to DropSouthPole

**Pull request merged**